### DISCUSSION TITLE
Add react-email's email preview

### DISCUSSION THREAD

**Discussion by andrecasal** [epicweb-dev/epic-stack#222] [Ideas]
We can preview our email's design with `react-email`'s CLI.

Cons:
- There's currently [this issue](https://github.com/resendlabs/react-email/issues/728), but I suspect it's an easy fix.
- It installs a slugish next.js server at `.react-email` to do this 😅 I expect this to become snappy over time.

Pros:
- This would allow us to preview and quickly iterate on the email design

I'd be happy to create a PR for this.

**Comment by kentcdodds** [comment]
I would much rather see movement on this: https://github.com/resendlabs/react-email/discussions/721

I *really* don't want to have to include all of next and its dependencies in the `node_modules` folks have to install when they start development of an Epic Stack app. I have similar feelings about Storybook.

**Reply by andrecasal** [reply]
Great idea 👌

**Reply by moishinetzer** [reply]
@kentcdodds Storybook works great when using the remix-run/testing package. Its very reliable and covers all of remix utilities. I'd be happy to create a sample PR to show what this would look like if you want?

I've set up storybook for my company that uses a remix project and it was an amazing productivity boost for our team.

**Reply by kentcdodds** [reply]
Thanks! An example would be best if you're willing to put one together following these instructions: https://github.com/epicweb-dev/epic-stack/blob/main/docs/examples.md
